apps/explorer/src/ui/StatAmount.tsx

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { SUI_TYPE_ARG, Coin } from '@mysten/sui.js';
+
+import { numberSuffix } from '../utils/numberUtil';
+
+import { CoinFormat, formatBalance } from '~/hooks/useFormatCoin';
```

Comment:
This component probably shouldn’t use the coin formatting hook. In general we want these to be entirely prop driven, so that they can be more easily composed.

Reply:
@Jordan-Mysten qq is there a reason why the text or header component do not accept optional color prop? Right now we have to wrap text or header component with html element to change the color

Reply:
There’s an open issue for it, I agree we should update it to accept a color out of a known set of colors.